packages/rancher-monitoring/rancher-node-exporter/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,27 +1,19 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -3,14 +3,8 @@
-   artifacthub.io/links: |
+@@ -1,6 +1,6 @@
+ annotations:
+   artifacthub.io/license: Apache-2.0
+-  artifacthub.io/links: |
++  artifacthub.io/links: |-
      - name: Chart Source
        url: https://github.com/prometheus-community/helm-charts
--apiVersion: v2
- appVersion: 1.6.0
--description: A Helm chart for prometheus node-exporter
- home: https://github.com/prometheus/node_exporter/
--keywords:
--- node-exporter
--- prometheus
--- exporter
- maintainers:
- - email: gianrubio@gmail.com
-   name: gianrubio
-@@ -18,8 +12,7 @@
```

Comment:
Looks like this removal causes an invalid Chart.yaml to be generated? See below where 
```
apiVersion: v2
   name: zanhsieh
 - email: rootsandtrees@posteo.de
   name: zeritti
```

Reply:
Correct me if you're seeing something different, but it renders fine locally for me. Could be confusing with reading a diff of a diff?

Reply:
yep